packages/beacon-node/src/node/nodejs.ts

```diff
@@ -213,6 +213,14 @@ export class BeaconNode {
     // Load persisted data from disk to in-memory caches
     await chain.loadFromDisk();
 
+    if (opts.chain.persistInvalidSszObjects && opts.chain.persistInvalidSszObjectsRetention !== 0) {
+      void chain.prunePastInvalidSszObjects();
+      const pruneIntervalId = setInterval(() => chain.prunePastInvalidSszObjects(), 1000 * 60 * 60);
```

Comment:
highly opinionated but I can't intuitively read this, changing the order of units would be nice

```suggestion
      const pruneIntervalId = setInterval(() => chain.prunePastInvalidSszObjects(),  60 * 60 * 1000);
```